example/exampleUser/plugin.js

```diff
@@ -21,14 +21,16 @@
  *****************************************************************************/
 
 import ExampleUserProvider from './ExampleUserProvider';
+const AUTO_LOGIN_USER = 'guest';
+const STATUS_ROLES = ['test-role-1', 'test-role-2', 'test-role-3'];
```

Comment:
We should add a mix of status role, and non-status roles here. And maybe name them something more evocative? E.g., flight-director, driver, and observer? In this case, observer wouldn't be able to status.